targets/TARGET_STM/rtc_api_hal.h

```diff
@@ -47,7 +66,11 @@ extern "C" {
 #if DEVICE_LPTICKER && !MBED_CONF_TARGET_LPTICKER_LPTIM
 /* PREDIV_A : 7-bit asynchronous prescaler */
 /* PREDIV_A is set to set LPTICKER frequency to RTC_CLOCK/4 */
+#if (MBED_CONF_TARGET_RTC_CLOCK_SOURCE == USE_RTC_CLK_HSE)
+#define PREDIV_A_VALUE 124
+#else
```

Comment:
Not in line with targets.json ?
"LPTICKER is not available for HSE and should be removed from the target configuration"

Reply:
Correct, this part is not necessary. I will remove it.